SummerProject/static/src/components/Footer/index.js

```diff
@@ -47,10 +47,10 @@ class Footer extends React.Component {
                                         <div className="icons">
                                             <div className="Sign-up">Sign up by</div>
                                             <div className="facebook-icon">
-                                                <img src="/static/src/components/Footer/imgs/fb.svg" alt="facebook"/>
+                                                <p><a href="#"><img src="/static/src/components/imgs/fb.svg" alt="facebook"/></a></p>
                                             </div>
                                             <div className="google-icon">
-                                                <img src="/static/src/components/Footer/imgs/gg.svg" alt="google"/>
+                                                <p><a href="#"><img src="/static/src/components/imgs/gg.svg" alt="google"/></a></p>
```

Comment:
Why did you wrap the tag **_a_** by tag _**p**_?

Reply:
found in tutorial example like that.

Reply:
U can remove tag **_p_** and nothing will change